### DISCUSSION TITLE
Niethammer Innovation Award 2023

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#352] [Announcements]
@Akaflieg-Freiburg/enroute We are glad to announce that the flight club [Akaflieg Freiburg](https://akaflieg-freiburg.de) has received the renowned Niethammer Innovation Award 2023. The award recognizes our work on **Enroute Flight Navigation**. But above all, we owe this prize to our many contributors and supporters. None of this would have been possible without the incredible work of the communities at [openAIP](https://www.openaip.net) and [open flightmaps](https://www.openflightmaps.org), who supply us with aviation data. 

**Thanks to all of you!**

We will use the award money to finance our infrastructure and purchase urgently needed development hardware.

**Comment by TTC70** [comment]
Stefan,
Happy to hear that, and even happier to be part of the project.
Cheers and keep it going 💪💪💪

Antonio

Il Lun 30 Ott 2023, 13:03 Stefan Kebekus ***@***.***> ha
scritto:

**Comment by Lukesw74** [comment]
Hello Stefan

cool, very good news ... 

Ciao
Luca
